src/App.js

```diff
@@ -10,29 +10,35 @@ class App extends Component {
     super();
     this.state = {
       movies:[],
-       error:null
+
+      error:null,
+      selected: null
+
     }
     
   }
 
   showMore = (id) => {
-    const selectedMovie = this.state.movies.filter(movie => movie.id === id)
-    this.setState({movies: selectedMovie})
-    // render() {
-    //   return (
-    //   <SingleView />
-    //   )
-    // }
+    const selectedMovie = this.state.movies.find(movie => movie.id === id)
+    this.setState({ selected: selectedMovie })
+  }
+
+  showAll = () => {
+    console.log('SHIT SHIT SHIT')
+    this.setState({ selected: null })
   }
 
   
 
   componentDidMount = () => {
-    console.log('App, componentDidMount')
+    // console.log('App, componentDidMount')
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
       .then((response) => response.json())
```

Comment:
You'll want to handle responses that are not ok by throwing an Error (I know this wasnt an edit it this PR but noticed so I wanted to comment on that anyway)